docs/src/get_started.md

```diff
@@ -0,0 +1,202 @@
+# Sampling from a multivariate Gaussian using NUTS
+
+In this get-start section, we demonstrate a minimal example of sampling from a multivariate Gaussian (10-dimensional) using the no U-turn sampler (NUTS). Below we describe the major components of the Hamiltonian system which are essential to sample using this approach:
```

Comment:
```suggestion
In this section, we demonstrate a minimal example of sampling from a multivariate Gaussian (10-dimensional) using the No U-Turn Sampler (NUTS). Below we describe the major components of the Hamiltonian system which are essential to sample using this approach:
```
If you do want to keep the word, it should probably be "get-started", but I think it's unnecessary.